### DISCUSSION TITLE
Can't send any commands to car? New Tesla 2024

### DISCUSSION THREAD

**Discussion by Joe123-B** [alandtse/tesla#871] [Q&A] [answered]
Hi, first of all thank you for this great integration! 
I wanted to know if it's just me or is sending commands to the car ( ex: horn, start charging, changing charging amps etc...) don't actually work? I own a new 2024 Tesla.
The integration provides very valuable data from the car so much appreciate the efforts that have gone in to build and enable this integration!!!

thank you

**Comment by Cowtschi** [comment]
Hello, I also can't send any commands to the vehicle, receiving works.
M3 and MY, both 2023.
LG

**Comment by Echotrouble** [comment]
I have a similar issue. Do you get an error tho?

**Reply by Joe123-B** [reply]
I do, i get the following error message:
![Screenshot 2024-02-26 223107](https://github.com/alandtse/tesla/assets/52842345/bcd61f74-e5d9-4663-a318-c7e8618596eb)

**Comment by baylanger** [comment, marked as answer]
There’s already opened issues on this topic. Go to Issues and search for “command” or just scroll down, it’s on the first page #743 and #774.

**Reply by Joe123-B** [reply]
yes I just saw, but couldn't make out whether this is a known issue for everyone? and if it is what potential solutions out there ?

**Reply by skipishere** [reply]
It's a problem for most newly bought cars, in theory we're currently waiting in borrowed time for those that it still works for (me included). The issues linked have some discussions about alternatives but at the moment we're waiting to see if Tesla will make it easy to work in a way that doesn't require everyone to create a developer account or host a fleet endpoint online

**Reply by Joe123-B** [reply]
ah ok got it, thank you for your reply and clarification. I'll wait then until Tesla decides to make things easier. not looking to create a dev account or host a fleet endpoint online. thanks again and please keep us posted.

All the best
